imagersite/imagersite/static/admin/css/rtl.css

```diff
@@ -0,0 +1,256 @@
+body {
```

Comment:
It's unfortunate that you have the collected static files from Django's own admin in your static files directory.  I imagine that now that projects are over, you realize why this happened, and what to do about it.  If not, ask.  It's not good.